src/swarm/neo/client/RequestSet.d

```diff
@@ -606,7 +606,13 @@ public final class RequestSet: IRequestSet
         private void reset ( )
         out
         {
-            assert(this); // invariant
+            if (!this.id)
```

Comment:
Wait, what's the reason for this?

Reply:
Thanks for pointing out. It seems like I missed it. I'll do another sweep.

Reply:
Ah, I see. I copied the contents of the invariant into the out block, so that checks specified here are not removed.

Reply:
Why though? (This isn't what the assert/enforce/verify doc recommends doing. Unless I'm going mad.)